ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -185,7 +223,4 @@ fi
 # FIXME: Remove when https://github.com/kata-containers/kata-containers/pull/8417 is resolved
 # Selinux context is currently not handled by kata-deploy
 oc apply -f ${deployments_dir}/relabel_selinux.yaml
-( for I in $(seq 30); do
-	sleep 10
-	oc logs -n kube-system ds/relabel-selinux-daemonset | grep "NSENTER_FINISHED_WITH:" && exit
-done ) || { echo "Selinux relabel failed, check the logs"; exit -1; }
+wait_for_app_pods_message restorecon "NSENTER_FINISHED_WITH:" 120 "kube-system" || echo "Failed to treat selinux, proceeding anyway..."
```

Comment:
Wondering why not `exit -1` like before (`{ echo "Selinux relabel failed, check the logs"; exit -1; }`)

Reply:
I'm fine either way, in this way I consider such failure more like a warning and I'd prefer having testing performed with chance of looking into logs for failures rather than not performed tests because of some bad exit code.